docset/winserver2019-ps/failoverclusters/New-Cluster.md

```diff
@@ -168,6 +168,29 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ManagementPointNetworkType
+Specifies the network configuration used to determine IP address settings.
+
+The acceptable values for this parameter are:
+
+```yaml
+Singleton: Uses the traditional method of DHCP or static IP address.
+Distributed: Use a Distributed Network Name by using Node IP addresses.
+Automatic: Uses detection to determine the appropriate setting. If SQL Server is running in Azure, uses Distributed. If SQL Server is running on-premises, uses Singleton (default setting).
+```
+
+```yaml
+Type: AdminAccessPointResType
+Parameter Sets: (All)
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
```

Comment:
```suggestion
Default value: Singleton
```

Double check this information to ensure accuracy. I'm making this changes based on your changes in the list of values above.

Reply:
When I used PlatyPS for WS2022, the default value was None.

Reply:
I believe that Automatic is the default value for this parameter, according to the few sources I've seen.